WordPressVIPMinimum/Tests/Files/IncludingFileUnitTest.inc

```diff
@@ -10,4 +10,14 @@ include ( MY_CONSTANT . "my_file.php" ); // Warning.
 
 require_once ( MY_CONSTANT . "my_file.php" ); // Warning.
 
-include( locate_template('index-loop.php') ); // Ok.
\ No newline at end of file
+include( locate_template('index-loop.php') ); // Ok.
+
+include( 'http://www.google.com/bad_file.php' ); // Bad.
```

Comment:
Can you update the comments to explain _why_ each of these test cases are OK/Bad/Warning?